### DISCUSSION TITLE
question about disk available of monitor

### DISCUSSION THREAD

**Discussion by jinlinyang2024** [apache/dolphinscheduler#13765] [Q&A]
I have deployed DolphinScheduler on my Ali cloud server with pseudo-cluster deployment mode. My Aliyun cloud server has two cloud disks, one is the system disk (40G) and the other is the data disk (500G). However, I can only see the "disk available" displayed in the monitoring center is the system disk (40G). How to operate or what kind of way to operate, so that I can see the data disk or all disk's "available disk capacity" indicator? 

dolphinscheduler磁盘可用容量(Disk Available)
![image](https://user-images.githubusercontent.com/124127997/225802168-306f2cc1-9c61-466f-82e2-910726f214fc.png)

真实磁盘使用情况(Actual disk available:)：
![image](https://user-images.githubusercontent.com/124127997/225802198-f5b7ac09-7d40-420f-89a4-52d6350f1c59.png)

**Comment by Tandoy** [comment]
1.This monitoring indicator is written to the zookeeper while maintaining the heartbeat. You can obtain it from the zk cli by using the following command：
`get /dolphinscheduler_dev/nodes/master/ip:5679`

2.The following source code can be modified
# https://github.com/apache/dolphinscheduler/blob/69e744961b489c9553e656b8d1f6247b46884e05/dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java#L119

**Comment by jinlinyang2024** [comment]
Sorry for replying so late. Thank you very much for your answer. I will try the solution you provided. Thank you so much!!!
